File: src/test/java/parse/GetAvailableVoices.java
package parse;

import util.jAdapterForNativeTTS.engines.*;
import util.jAdapterForNativeTTS.engines.exceptions.SpeechEngineCreationException;
import java.util.List;

public class GetAvailableVoices {
  public static void main(String[] args) throws SpeechEngineCreationException {
    List<Voice> voices = SpeechEngineNative.getInstance().getAvailableVoices();
    for (Voice voice : voices) {
      System.out.println(voice);
    }
  }
}

File: src/test/java/parse/GetAvailableVoices.kt
package parse

import io.github.jonelo.tts.engines.SpeechEngineNative
import org.apache.logging.log4j.LogManager

fun main() {
  SpeechEngineNative.getInstance().availableVoices.forEach {
    LogManager.getLogger().info(it)
  }
}